components/esp_rom/patches/esp_rom_hp_regi2c_esp32c6.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * SPDX-FileCopyrightText: 2019-2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "esp_rom_sys.h"
```

Comment:
just a note to check if we use or not (and should) this file

Reply:
I think we should use all patches when available.